docs/reference/esql/functions/README.md

```diff
@@ -0,0 +1,21 @@
+The files in these subdirectories and generated by ESQL's test suite:
```

Comment:
```suggestion
The files in these subdirectories are generated by ESQL's test suite:
```

Reply:
I hadn't seen this before the merge landed. Here's the fix: https://github.com/elastic/elasticsearch/commit/d6d1edd52959f358ce1df438d01ddf03ef8b5c21